{
    "version": "https://jsonfeed.org/version/1",
    "title": "Security Advisory for Python packages hosted at PyPI.org",
    "home_page_url": "https://github.com/advisories?query=type%3Areviewed+ecosystem%3Apip",
    "feed_url": "https://azu.github.io/github-advisory-database-rss/pip.json",
    "description": "Security Advisory for Python packages hosted at PyPI.org on GitHub",
    "items": [
        {
            "content_html": "<p>SQLAlchemy 1.2.17 has SQL Injection when the group_by parameter can be controlled.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2019-7548\">https://nvd.nist.gov/vuln/detail/CVE-2019-7548</a></li>\n<li><a href=\"https://github.com/sqlalchemy/sqlalchemy/issues/4481#issuecomment-461204518\">https://github.com/sqlalchemy/sqlalchemy/issues/4481#issuecomment-461204518</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0981\">https://access.redhat.com/errata/RHSA-2019:0981</a></li>\n<li><a href=\"https://access.redhat.com/errata/RHSA-2019:0984\">https://access.redhat.com/errata/RHSA-2019:0984</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-38fc-9xqv-7f7q\">https://github.com/advisories/GHSA-38fc-9xqv-7f7q</a></li>\n<li><a href=\"https://github.com/no-security/sqlalchemy_test\">https://github.com/no-security/sqlalchemy_test</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2019/03/msg00020.html\">https://lists.debian.org/debian-lts-announce/2019/03/msg00020.html</a></li>\n<li><a href=\"https://www.oracle.com/security-alerts/cpujan2021.html\">https://www.oracle.com/security-alerts/cpujan2021.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-security-announce/2019-08/msg00087.html\">http://lists.opensuse.org/opensuse-security-announce/2019-08/msg00087.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00010.html\">http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00010.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00016.html\">http://lists.opensuse.org/opensuse-security-announce/2019-09/msg00016.html</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2021/11/msg00005.html\">https://lists.debian.org/debian-lts-announce/2021/11/msg00005.html</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-38fc-9xqv-7f7q",
            "title": "[SQLAlchemy] SQLAlchemy is vulnerable to SQL Injection via group_by parameter ",
            "date_modified": "2023-09-01T09:56:01.000Z",
            "date_published": "2019-04-16T15:50:39.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-38fc-9xqv-7f7q"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>The Python websockets library version 4 contains a CWE-409: Improper Handling of Highly Compressed Data (Data Amplification) vulnerability in Servers and clients, unless configured with compression=None that can result in Denial of Service by memory exhaustion. This attack appears to be exploitable via sending a specially crafted frame on an established connection. This vulnerability appears to have been fixed in version 5.0</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-1000518\">https://nvd.nist.gov/vuln/detail/CVE-2018-1000518</a></li>\n<li><a href=\"https://github.com/aaugustin/websockets/pull/407\">https://github.com/aaugustin/websockets/pull/407</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-6g87-ff9q-v847\">https://github.com/advisories/GHSA-6g87-ff9q-v847</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-6g87-ff9q-v847",
            "title": "[websockets] websockets is vulnerable to denial of service by memory exhaustion",
            "date_modified": "2023-09-01T09:53:43.000Z",
            "date_published": "2018-09-17T20:46:52.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-6g87-ff9q-v847"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>CRLF Injection in pypiserver 1.2.5 and below allows attackers to set arbitrary HTTP headers and possibly conduct XSS attacks via a <code>%0d%0a</code> in a URI.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2019-6802\">https://nvd.nist.gov/vuln/detail/CVE-2019-6802</a></li>\n<li><a href=\"https://github.com/pypiserver/pypiserver/issues/237\">https://github.com/pypiserver/pypiserver/issues/237</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-mh24-7wvg-v88g\">https://github.com/advisories/GHSA-mh24-7wvg-v88g</a></li>\n<li><a href=\"https://github.com/pypiserver/pypiserver/commit/1375a67c55a9b8d4619df30d2a1c0b239d7357e6\">https://github.com/pypiserver/pypiserver/commit/1375a67c55a9b8d4619df30d2a1c0b239d7357e6</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-mh24-7wvg-v88g",
            "title": "[pypiserver] CRLF Injection in pypiserver",
            "date_modified": "2023-09-01T09:46:46.000Z",
            "date_published": "2019-01-30T20:56:26.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-mh24-7wvg-v88g"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An exploitable vulnerability exists in the YAML parsing functionality in config.py in Confire 0.2.0. Due to the user-specific configuration being loaded from \"~/.confire.yaml\" using the yaml.load function, a YAML parser can execute arbitrary Python commands resulting in command execution. An attacker can insert Python into loaded YAML to trigger this vulnerability.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-16763\">https://nvd.nist.gov/vuln/detail/CVE-2017-16763</a></li>\n<li><a href=\"https://github.com/bbengfort/confire/issues/24\">https://github.com/bbengfort/confire/issues/24</a></li>\n<li><a href=\"https://github.com/bbengfort/confire/commit/8cc86a5ec2327e070f1d576d61bbaadf861597ea\">https://github.com/bbengfort/confire/commit/8cc86a5ec2327e070f1d576d61bbaadf861597ea</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-m85c-9mf8-m2m6\">https://github.com/advisories/GHSA-m85c-9mf8-m2m6</a></li>\n<li><a href=\"https://joel-malwarebenchmark.github.io/blog/2017/11/12/cve-2017-16763-configure-loaded-through-confire/\">https://joel-malwarebenchmark.github.io/blog/2017/11/12/cve-2017-16763-configure-loaded-through-confire/</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-m85c-9mf8-m2m6",
            "title": "[confire] Unsafe deserialization in confire",
            "date_modified": "2023-09-01T09:38:00.000Z",
            "date_published": "2018-07-18T18:28:26.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-m85c-9mf8-m2m6"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>An exploitable vulnerability exists in the YAML loading functionality of util.py in OwlMixin before 2.0.0a12. A \"Load YAML\" string or file (aka load_yaml or load_yamlf) can execute arbitrary Python commands resulting in command execution because load is used where safe_load should have been used. An attacker can insert Python into loaded YAML to trigger this vulnerability.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-16618\">https://nvd.nist.gov/vuln/detail/CVE-2017-16618</a></li>\n<li><a href=\"https://github.com/tadashi-aikawa/owlmixin/issues/12\">https://github.com/tadashi-aikawa/owlmixin/issues/12</a></li>\n<li><a href=\"https://github.com/tadashi-aikawa/owlmixin/commit/5d0575303f6df869a515ced4285f24ba721e0d4e\">https://github.com/tadashi-aikawa/owlmixin/commit/5d0575303f6df869a515ced4285f24ba721e0d4e</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-ccmq-qvcp-5mrm\">https://github.com/advisories/GHSA-ccmq-qvcp-5mrm</a></li>\n<li><a href=\"https://joel-malwarebenchmark.github.io/blog/2017/11/08/cve-2017-16618-convert-through-owlmixin/\">https://joel-malwarebenchmark.github.io/blog/2017/11/08/cve-2017-16618-convert-through-owlmixin/</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-ccmq-qvcp-5mrm",
            "title": "[owlmixin] Unsafe deserialization in owlmixin",
            "date_modified": "2023-09-01T09:34:02.000Z",
            "date_published": "2018-07-13T16:01:12.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-ccmq-qvcp-5mrm"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>modulemd 1.3.1 and earlier uses an unsafe function for processing externally provided data, leading to remote code execution.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2017-1002157\">https://nvd.nist.gov/vuln/detail/CVE-2017-1002157</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-jhjh-ghwx-6h7r\">https://github.com/advisories/GHSA-jhjh-ghwx-6h7r</a></li>\n<li><a href=\"https://pagure.io/modulemd/issue/55\">https://pagure.io/modulemd/issue/55</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-jhjh-ghwx-6h7r",
            "title": "[modulemd] modulemd uses an unsafe function for processing externally provided data",
            "date_modified": "2023-09-01T09:30:30.000Z",
            "date_published": "2019-01-17T13:56:18.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-jhjh-ghwx-6h7r"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Poetry prior to v1.1.9 was discovered to contain an untrusted search path which causes the application to behave in unexpected ways when users execute Poetry commands in a directory containing malicious content. This vulnerability occurs when the application is ran on Windows OS.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2022-26184\">https://nvd.nist.gov/vuln/detail/CVE-2022-26184</a></li>\n<li><a href=\"https://github.com/python-poetry/poetry-core/pull/205/commits/fa9cb6f358ae840885c700f954317f34838caba7\">https://github.com/python-poetry/poetry-core/pull/205/commits/fa9cb6f358ae840885c700f954317f34838caba7</a></li>\n<li><a href=\"https://github.com/python-poetry/poetry/releases/tag/1.1.9\">https://github.com/python-poetry/poetry/releases/tag/1.1.9</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/poetry/PYSEC-2022-234.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/poetry/PYSEC-2022-234.yaml</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-xr2c-5w89-63pv\">https://github.com/advisories/GHSA-xr2c-5w89-63pv</a></li>\n<li><a href=\"https://github.com/python-poetry/poetry-core/commit/1e1a109a1009daaab2367ce90c997f0cbbb0c1d1\">https://github.com/python-poetry/poetry-core/commit/1e1a109a1009daaab2367ce90c997f0cbbb0c1d1</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-xr2c-5w89-63pv",
            "title": "[poetry] Poetry before v1.1.9 contains Untrusted Search Path",
            "date_modified": "2023-09-01T05:02:40.000Z",
            "date_published": "2022-03-23T00:00:24.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-xr2c-5w89-63pv"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>util/emailutils.py in Matrix Sydent before 1.0.2 mishandles registration restrictions that are based on e-mail domain, if the allowed_local_3pids option is enabled. This occurs because of potentially unwanted behavior in Python, in which an email.utils.parseaddr call on <a href=\"mailto:user@bad.example.net\">user@bad.example.net</a>@good.example.com returns the <a href=\"mailto:user@bad.example.net\">user@bad.example.net</a> substring.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2019-11340\">https://nvd.nist.gov/vuln/detail/CVE-2019-11340</a></li>\n<li><a href=\"https://github.com/matrix-org/sydent/commit/4e1cfff53429c49c87d5c457a18ed435520044fc\">https://github.com/matrix-org/sydent/commit/4e1cfff53429c49c87d5c457a18ed435520044fc</a></li>\n<li><a href=\"https://github.com/matrix-org/sydent/compare/7c002cd...09278fb\">https://github.com/matrix-org/sydent/compare/7c002cd...09278fb</a></li>\n<li><a href=\"https://matrix.org/blog/2019/04/18/security-update-sydent-1-0-2/\">https://matrix.org/blog/2019/04/18/security-update-sydent-1-0-2/</a></li>\n<li><a href=\"https://twitter.com/matrixdotorg/status/1118934335963500545\">https://twitter.com/matrixdotorg/status/1118934335963500545</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-q9h8-gpw5-c95c\">https://github.com/advisories/GHSA-q9h8-gpw5-c95c</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-q9h8-gpw5-c95c",
            "title": "[matrix-sydent] Matrix Sydent mishandles emails",
            "date_modified": "2023-09-01T00:00:57.000Z",
            "date_published": "2022-05-24T16:43:57.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-q9h8-gpw5-c95c"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in ansible. Credentials, such as secrets, are being disclosed in console log by default and not protected by no_log feature when using those modules. An attacker can take advantage of this information to steal those credentials. The highest threat from this vulnerability is to data confidentiality.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-20191\">https://nvd.nist.gov/vuln/detail/CVE-2021-20191</a></li>\n<li><a href=\"https://access.redhat.com/security/cve/cve-2021-20191\">https://access.redhat.com/security/cve/cve-2021-20191</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1916813\">https://bugzilla.redhat.com/show_bug.cgi?id=1916813</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73488\">https://github.com/ansible/ansible/pull/73488</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73489\">https://github.com/ansible/ansible/pull/73489</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-8f4m-hccc-8qph\">https://github.com/advisories/GHSA-8f4m-hccc-8qph</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-8f4m-hccc-8qph",
            "title": "[ansible] Insertion of Sensitive Information into Log File in ansible",
            "date_modified": "2023-08-31T21:50:10.000Z",
            "date_published": "2021-06-01T21:38:33.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-8f4m-hccc-8qph"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in ansible. Credentials, such as secrets, are being disclosed in console log by default and not protected by no_log feature when using those modules. An attacker can take advantage of this information to steal those credentials. The highest threat from this vulnerability is to data confidentiality.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-20191\">https://nvd.nist.gov/vuln/detail/CVE-2021-20191</a></li>\n<li><a href=\"https://access.redhat.com/security/cve/cve-2021-20191\">https://access.redhat.com/security/cve/cve-2021-20191</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1916813\">https://bugzilla.redhat.com/show_bug.cgi?id=1916813</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73488\">https://github.com/ansible/ansible/pull/73488</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73489\">https://github.com/ansible/ansible/pull/73489</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-8f4m-hccc-8qph\">https://github.com/advisories/GHSA-8f4m-hccc-8qph</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-8f4m-hccc-8qph",
            "title": "[ansible] Insertion of Sensitive Information into Log File in ansible",
            "date_modified": "2023-08-31T21:50:10.000Z",
            "date_published": "2021-06-01T21:38:33.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-8f4m-hccc-8qph"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in ansible. Credentials, such as secrets, are being disclosed in console log by default and not protected by no_log feature when using those modules. An attacker can take advantage of this information to steal those credentials. The highest threat from this vulnerability is to data confidentiality.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-20191\">https://nvd.nist.gov/vuln/detail/CVE-2021-20191</a></li>\n<li><a href=\"https://access.redhat.com/security/cve/cve-2021-20191\">https://access.redhat.com/security/cve/cve-2021-20191</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1916813\">https://bugzilla.redhat.com/show_bug.cgi?id=1916813</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73488\">https://github.com/ansible/ansible/pull/73488</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73489\">https://github.com/ansible/ansible/pull/73489</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-8f4m-hccc-8qph\">https://github.com/advisories/GHSA-8f4m-hccc-8qph</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-8f4m-hccc-8qph",
            "title": "[ansible] Insertion of Sensitive Information into Log File in ansible",
            "date_modified": "2023-08-31T21:50:10.000Z",
            "date_published": "2021-06-01T21:38:33.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-8f4m-hccc-8qph"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in the Ansible Engine 2.9.18, where sensitive info is not masked by default and is not protected by the <code>no_log</code> feature when using the sub-option feature of the basic.py module. This flaw allows an attacker to obtain sensitive information. The highest threat from this vulnerability is to confidentiality.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-20228\">https://nvd.nist.gov/vuln/detail/CVE-2021-20228</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73487\">https://github.com/ansible/ansible/pull/73487</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73492\">https://github.com/ansible/ansible/pull/73492</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73493\">https://github.com/ansible/ansible/pull/73493</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73494\">https://github.com/ansible/ansible/pull/73494</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1925002\">https://bugzilla.redhat.com/show_bug.cgi?id=1925002</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-5rrg-rr89-x9mv\">https://github.com/advisories/GHSA-5rrg-rr89-x9mv</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-1.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-1.yaml</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv",
            "title": "[ansible] Ansible Exposes Sensitive Information",
            "date_modified": "2023-08-31T21:49:02.000Z",
            "date_published": "2022-05-25T19:22:34.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in the Ansible Engine 2.9.18, where sensitive info is not masked by default and is not protected by the <code>no_log</code> feature when using the sub-option feature of the basic.py module. This flaw allows an attacker to obtain sensitive information. The highest threat from this vulnerability is to confidentiality.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-20228\">https://nvd.nist.gov/vuln/detail/CVE-2021-20228</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73487\">https://github.com/ansible/ansible/pull/73487</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73492\">https://github.com/ansible/ansible/pull/73492</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73493\">https://github.com/ansible/ansible/pull/73493</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73494\">https://github.com/ansible/ansible/pull/73494</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1925002\">https://bugzilla.redhat.com/show_bug.cgi?id=1925002</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-5rrg-rr89-x9mv\">https://github.com/advisories/GHSA-5rrg-rr89-x9mv</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-1.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-1.yaml</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv",
            "title": "[ansible] Ansible Exposes Sensitive Information",
            "date_modified": "2023-08-31T21:49:02.000Z",
            "date_published": "2022-05-25T19:22:34.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in the Ansible Engine 2.9.18, where sensitive info is not masked by default and is not protected by the <code>no_log</code> feature when using the sub-option feature of the basic.py module. This flaw allows an attacker to obtain sensitive information. The highest threat from this vulnerability is to confidentiality.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2021-20228\">https://nvd.nist.gov/vuln/detail/CVE-2021-20228</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73487\">https://github.com/ansible/ansible/pull/73487</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73492\">https://github.com/ansible/ansible/pull/73492</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73493\">https://github.com/ansible/ansible/pull/73493</a></li>\n<li><a href=\"https://github.com/ansible/ansible/pull/73494\">https://github.com/ansible/ansible/pull/73494</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1925002\">https://bugzilla.redhat.com/show_bug.cgi?id=1925002</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-5rrg-rr89-x9mv\">https://github.com/advisories/GHSA-5rrg-rr89-x9mv</a></li>\n<li><a href=\"https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-1.yaml\">https://github.com/pypa/advisory-database/tree/main/vulns/ansible/PYSEC-2021-1.yaml</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv",
            "title": "[ansible] Ansible Exposes Sensitive Information",
            "date_modified": "2023-08-31T21:49:02.000Z",
            "date_published": "2022-05-25T19:22:34.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-5rrg-rr89-x9mv"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in the Ansible Engine, in ansible-engine 2.8.x before 2.8.15 and ansible-engine 2.9.x before 2.9.13, when installing packages using the dnf module. GPG signatures are ignored during installation even when <code>disable_gpg_check</code> is set to <code>False</code>, which is the default behavior. This flaw leads to malicious packages being installed on the system and arbitrary code executed via package installation scripts. The highest threat from this vulnerability is to integrity and system availability.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-14365\">https://nvd.nist.gov/vuln/detail/CVE-2020-14365</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1869154\">https://bugzilla.redhat.com/show_bug.cgi?id=1869154</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/9bea33ffa3be3d64827f59882d95b817cfab9b7e\">https://github.com/ansible/ansible/commit/9bea33ffa3be3d64827f59882d95b817cfab9b7e</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-m429-fhmv-c6q2\">https://github.com/advisories/GHSA-m429-fhmv-c6q2</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-m429-fhmv-c6q2",
            "title": "[ansible] Improper Verification of Cryptographic Signature in ansible",
            "date_modified": "2023-08-31T21:46:31.000Z",
            "date_published": "2021-04-20T16:44:07.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-m429-fhmv-c6q2"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>A flaw was found in the Ansible Engine, in ansible-engine 2.8.x before 2.8.15 and ansible-engine 2.9.x before 2.9.13, when installing packages using the dnf module. GPG signatures are ignored during installation even when <code>disable_gpg_check</code> is set to <code>False</code>, which is the default behavior. This flaw leads to malicious packages being installed on the system and arbitrary code executed via package installation scripts. The highest threat from this vulnerability is to integrity and system availability.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2020-14365\">https://nvd.nist.gov/vuln/detail/CVE-2020-14365</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=1869154\">https://bugzilla.redhat.com/show_bug.cgi?id=1869154</a></li>\n<li><a href=\"https://www.debian.org/security/2021/dsa-4950\">https://www.debian.org/security/2021/dsa-4950</a></li>\n<li><a href=\"https://github.com/ansible/ansible/commit/9bea33ffa3be3d64827f59882d95b817cfab9b7e\">https://github.com/ansible/ansible/commit/9bea33ffa3be3d64827f59882d95b817cfab9b7e</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-m429-fhmv-c6q2\">https://github.com/advisories/GHSA-m429-fhmv-c6q2</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-m429-fhmv-c6q2",
            "title": "[ansible] Improper Verification of Cryptographic Signature in ansible",
            "date_modified": "2023-08-31T21:46:31.000Z",
            "date_published": "2021-04-20T16:44:07.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-m429-fhmv-c6q2"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>qutebrowser before version 1.4.1 is vulnerable to a cross-site request forgery flaw that allows websites to access <code>qute://*</code> URLs. A malicious website could exploit this to load a <code>qute://settings/set</code> URL, which then sets <code>editor.command</code> to a bash script, resulting in arbitrary code execution.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2018-10895\">https://nvd.nist.gov/vuln/detail/CVE-2018-10895</a></li>\n<li><a href=\"https://github.com/qutebrowser/qutebrowser/commit/43e58ac865ff862c2008c510fc5f7627e10b4660\">https://github.com/qutebrowser/qutebrowser/commit/43e58ac865ff862c2008c510fc5f7627e10b4660</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-wgmx-52ph-qqcw\">https://github.com/advisories/GHSA-wgmx-52ph-qqcw</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-wgmx-52ph-qqcw",
            "title": "[qutebrowser] Qutebrowser CSRF Vulnerability",
            "date_modified": "2023-08-31T21:42:55.000Z",
            "date_published": "2018-10-10T16:05:23.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-wgmx-52ph-qqcw"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>Ansible before 1.9.2 does not verify that the server hostname matches a domain name in the subject's Common Name (CN) or subjectAltName field of the X.509 certificate, which allows man-in-the-middle attackers to spoof SSL servers via an arbitrary valid certificate.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2015-3908\">https://nvd.nist.gov/vuln/detail/CVE-2015-3908</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-w64c-pxjj-h866\">https://github.com/advisories/GHSA-w64c-pxjj-h866</a></li>\n<li><a href=\"https://lists.debian.org/debian-lts-announce/2019/09/msg00016.html\">https://lists.debian.org/debian-lts-announce/2019/09/msg00016.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-07/msg00051.html\">http://lists.opensuse.org/opensuse-updates/2015-07/msg00051.html</a></li>\n<li><a href=\"http://lists.opensuse.org/opensuse-updates/2015-08/msg00029.html\">http://lists.opensuse.org/opensuse-updates/2015-08/msg00029.html</a></li>\n<li><a href=\"http://www.ansible.com/security\">http://www.ansible.com/security</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2015/07/14/4\">http://www.openwall.com/lists/oss-security/2015/07/14/4</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-w64c-pxjj-h866",
            "title": "[ansible] Ansible does not verify that the server hostname matches a domain name in certificates",
            "date_modified": "2023-08-31T21:41:25.000Z",
            "date_published": "2018-10-10T17:23:51.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-w64c-pxjj-h866"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>emitters.py in Django Piston before 0.2.3 and 0.2.x before 0.2.2.1 does not properly deserialize YAML data, which allows remote attackers to execute arbitrary Python code via vectors related to the yaml.load method.</p>\n<p>Django Tastypie has a very similar vulnerability.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2011-4103\">https://nvd.nist.gov/vuln/detail/CVE-2011-4103</a></li>\n<li><a href=\"https://bitbucket.org/jespern/django-piston/commits/91bdaec89543/\">https://bitbucket.org/jespern/django-piston/commits/91bdaec89543/</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=750658\">https://bugzilla.redhat.com/show_bug.cgi?id=750658</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-pvhp-v9qp-xf5r\">https://github.com/advisories/GHSA-pvhp-v9qp-xf5r</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2011/nov/01/piston-and-tastypie-security-releases/\">https://www.djangoproject.com/weblog/2011/nov/01/piston-and-tastypie-security-releases/</a></li>\n<li><a href=\"http://www.debian.org/security/2011/dsa-2344\">http://www.debian.org/security/2011/dsa-2344</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2011/11/01/10\">http://www.openwall.com/lists/oss-security/2011/11/01/10</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-pvhp-v9qp-xf5r",
            "title": "[django-piston] Django-piston and Django-tastypie do not properly deserialize YAML data",
            "date_modified": "2023-08-31T21:39:51.000Z",
            "date_published": "2018-07-23T19:50:48.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-pvhp-v9qp-xf5r"
            },
            "tags": [
                "severity"
            ]
        },
        {
            "content_html": "<p>emitters.py in Django Piston before 0.2.3 and 0.2.x before 0.2.2.1 does not properly deserialize YAML data, which allows remote attackers to execute arbitrary Python code via vectors related to the yaml.load method.</p>\n<p>Django Tastypie has a very similar vulnerability.</p>\n<h3 id=\"references\">References</h3>\n<ul>\n<li><a href=\"https://nvd.nist.gov/vuln/detail/CVE-2011-4103\">https://nvd.nist.gov/vuln/detail/CVE-2011-4103</a></li>\n<li><a href=\"https://bitbucket.org/jespern/django-piston/commits/91bdaec89543/\">https://bitbucket.org/jespern/django-piston/commits/91bdaec89543/</a></li>\n<li><a href=\"https://bugzilla.redhat.com/show_bug.cgi?id=750658\">https://bugzilla.redhat.com/show_bug.cgi?id=750658</a></li>\n<li><a href=\"https://github.com/advisories/GHSA-pvhp-v9qp-xf5r\">https://github.com/advisories/GHSA-pvhp-v9qp-xf5r</a></li>\n<li><a href=\"https://www.djangoproject.com/weblog/2011/nov/01/piston-and-tastypie-security-releases/\">https://www.djangoproject.com/weblog/2011/nov/01/piston-and-tastypie-security-releases/</a></li>\n<li><a href=\"http://www.debian.org/security/2011/dsa-2344\">http://www.debian.org/security/2011/dsa-2344</a></li>\n<li><a href=\"http://www.openwall.com/lists/oss-security/2011/11/01/10\">http://www.openwall.com/lists/oss-security/2011/11/01/10</a></li>\n</ul>\n",
            "url": "https://github.com/advisories/GHSA-pvhp-v9qp-xf5r",
            "title": "[django-piston] Django-piston and Django-tastypie do not properly deserialize YAML data",
            "date_modified": "2023-08-31T21:39:51.000Z",
            "date_published": "2018-07-23T19:50:48.000Z",
            "author": {
                "name": "GitHub",
                "url": "https://github.com/advisories/GHSA-pvhp-v9qp-xf5r"
            },
            "tags": [
                "severity"
            ]
        }
    ]
}